packaging/conda/meta.yaml

```diff
@@ -1,6 +1,8 @@
+{% set version = os.environ.get('KHIOPS_VERSION') %}
+
 package:
   name: khiops-binaries
-  version: {{ os.environ.get('KHIOPS_VERSION') }}
+  version: {{ version.replace("-", ".") }}
```

Comment:
I would write `{{ version.replace("-", "") }}` to remain consistent with the current behavior, and thus versioning scheme.